src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -239,7 +240,19 @@ private function handleThrowable(\Throwable $e, Request $request, int $type): Re
                 $response->setStatusCode($e->getStatusCode());
                 $response->headers->add($e->getHeaders());
             } else {
-                $response->setStatusCode(500);
+                $class = new \ReflectionClass($e);
```

Comment:
```suggestion
                $class = new \ReflectionClass($e, \ReflectionAttribute::IS_INSTANCEOF);
```

Reply:
Does that make sense? The `ReflectionClass` constructor does not take a second argument. 🤔

Reply:
I think this can be resolved as the `\ReflectionAttribute::IS_INSTANCEOF` flag is used in the `getAttributes` call. (Note that it wasn't being used when the comment was left by Kevin.)